integration_tests/models/source/databricks/snowplow_web_events_stg.sql

```diff
@@ -2,15 +2,161 @@
 with prep as (
 select
   *,
-  from_json(contexts_com_snowplowanalytics_snowplow_web_page_1_0_0, 'array<struct<id:string>>') as contexts_com_snowplowanalytics_snowplow_web_page_1
+  from_json(contexts_com_snowplowanalytics_snowplow_web_page_1_0_0, 'array<struct<id:string>>') as contexts_com_snowplowanalytics_snowplow_web_page_1,
+  from_json(unstruct_event_com_snowplowanalytics_snowplow_consent_preferences_1_0_0, 'array<struct<basis_for_processing:string, consent_scopes:string, consent_url:string, consent_version:string, domains_applied:string, event_type:string, gdpr_applies:string>>') as unstruct_event_com_snowplowanalytics_snowplow_consent_preferences_1,
+  from_json(unstruct_event_com_snowplowanalytics_snowplow_cmp_visible_1_0_0, 'array<struct<elapsed_time:string>>') as unstruct_event_com_snowplowanalytics_snowplow_cmp_visible_1
```

Comment:
Not sure I understand what you mean, the integration tests run correctly so I would say it is good as it is for now. Once we have real data in databricks and in case it fails the local tests I will get back to this and modify this as it would probably mean the underlying data types differ.